src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs

```diff
@@ -21,75 +22,88 @@ public CentOpsFixture(IConfiguration configuration, TestClients testClients)
 
             // Setup
             _configuration = configuration;
-            TestInstitutionName = $"TestInstitution{DateTime.UtcNow.ToString("yyyyMMddHHmmss", CultureInfo.CurrentCulture)}";
-            _institutionsUri = new Uri($"{_configuration["CentOpsUrl"]}/admin/institutions");
-            _participantsUri = new Uri($"{_configuration["CentOpsUrl"]}/admin/participants");
-            _testClients = testClients;
+            _useFixture = Convert.ToBoolean(_configuration["UseFixture"], CultureInfo.CurrentCulture);
```

Comment:
Please use `InvariantCulture`:
```suggestion
            _useFixture = Convert.ToBoolean(_configuration["UseFixture"], CultureInfo.InvariantCulture);
```

Reply:
Fixed, thanks